packages/lib/modules/pool/PoolDetail/PoolHeader/PoolHeader.tsx

```diff
@@ -29,7 +30,12 @@ export function PoolHeader() {
   const partnerRedirectDisclosure = useDisclosure()
   const isCowPool = isCowAmmPool(pool.type)
 
-  const isAddLiquidityBlocked = shouldBlockAddLiquidity(pool)
+  const shouldBlockCustom = isMaBeetsPool(pool.id)
```

Comment:
this hsould maybe somehow also move to the project config (inclusing the reason string)

Reply:
ok that kind of how i had it before :)
i've reverted it back with your suggestions